todoman/ui.py

```diff
@@ -24,19 +24,23 @@ class TodoEditor:
     The UI for a single todo entry.
     """
 
-    def __init__(self, todo, lists, formatter):
+    def __init__(self, todo, lists, formatter,current_list):
         """
         :param model.Todo todo: The todo object which will be edited.
         """
-
+        self.current_list = todo.list
         self.todo = todo
-        self.lists = lists
+        self.lists = list(lists)
         self.formatter = formatter
         self.saved = EditState.none
         self._loop = None
 
         self._msg_text = urwid.Text('')
 
+        urwid.RadioButton(self.lists, "label", state='first True', on_state_change=self.current_list+1, user_data=None)
```

Comment:
Not sure what you meant to do here by `+1`; `current_list` is a `List`, so you can't add +1 to it.